Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationPage.xaml.cs

```diff
@@ -70,15 +74,54 @@ public sealed partial class SystemInformationPage : Page
         // To get the time the app was previously launched, not including this instance
         public string LastLaunchTime => SystemInformation.LastLaunchTime.ToString(Culture.DateTimeFormat);
 
+        // To get the time the launch count was reset, not including this instance
+        public string LastResetTime
```

Comment:
Why DP now?

Reply:
We discussed this on the previous iteration.  Since these can be updated by the reset button, it seemed clearer to use DP for binding vs. implementing the INotifyPropertyChangedInterface.

Reply:
Well. I don't know what is the best approach. What's certain for me now is that implementing INotifyPropertyChanged brings some more code to the table while DP is pretty strainghtforward, But I suppose DP can be a PITA when you have to write more than 3/4 properties.